index.js

```diff
@@ -55,15 +55,30 @@ app.post('/', upload.single('thumb'), async(req, res, next) => {
   if (payload.event === 'media.play' || payload.event === 'media.rate') {
     if (image) {
       console.log('[REDIS]', `Using cached image ${key}`);
-    } else if (!image && req.file && req.file.buffer) {
-      console.log('[REDIS]', `Saving new image ${key}`);
-      image = await sharp(req.file.buffer)
-        .resize(75, 75)
-        .background('white')
-        .embed()
-        .toBuffer();
-
-      redis.set(key, image, 'EX', SEVEN_DAYS);
+    } else {
+      let buffer;
+      if (req.file && req.file.buffer) {
+        buffer = req.file.buffer;
+      } else if (payload.thumb) {
+        console.log('[REDIS]', `Retrieving image from  ${payload.thumb}`);
+        buffer = await request.get({
```

Comment:
We need to download and cache the image on the `play` event since the `scrobble` event does not send the `thumb` property. Considering this is in an `if` block for only `play` and `rate` events I suspect PMS does that same thing?